src/index-hindi.html

```diff
@@ -0,0 +1,655 @@
+
+          <p>Lastly,  I request the regulator to consider the submissions made to Question Nos. 14 and 15 in the Consultation on OTT Services (April, 2015) for the Consultation on Pricing Discrimination and come out with a definitive timetable for the conclusion of both consultation processes.</p>
+        </div>
+      </div>
+      <div class="clearfix"></div>
+      <div class="col-lg-12 text-center">
+        <button class="btn btn-info btn-lg sendResponse" id="bottomSend">Send your response</button>
+      </div>
+    </div>
+  </div>
+  <div class="container">
+    <hr>
+      <div class="row">
+        <div class="col-lg-4 col-md-4 col-sm-12">
+          <div class="panel panel-default">
+            <div class="panel-heading">
+              <h3 class="panel-title">What is Net Neutrality?</h3>
+            </div>
+            <div class="panel-body">
+              <a href="http://blog.savetheinternet.in/what-is-net-neutrality/">Read more...</a>
+            </div>
+          </div>
+        </div>
+        <div class="col-lg-4 col-md-4 col-sm-12">
+          <div class="panel panel-default">
+            <div class="panel-heading">
+              <h3 class="panel-title">Facebook's Free Basics</h3>
+            </div>
+            <div class="panel-body">
+              <a href="http://blog.savetheinternet.in/free-basics/">Our coverage of Free Basics</a>
+            </div>
+          </div>
+        </div>
+        <div class="col-lg-4 col-md-4 col-sm-12">
+          <div class="panel panel-default">
+            <div class="panel-heading">
+              <h3 class="panel-title">#SaveTheInternet Blog</h3>
+            </div>
+            <div class="panel-body">
+              <a href="http://blog.savetheinternet.in/">Updates from the SaveTheInternet team</a>
+            </div>
+          </div>
+        </div>
+      </div>
+  </div>
+  <footer>
+    <div class="container">
+      <div class="row">
+        <div class="col-lg-12 text-center">
+          <p>
+            <a href="https://github.com/netneutrality/" role="button" class="btn btn-sm btn-info">Made with ♥ by folks just like you</a>
+        </div>
+      </div>
+    </div>
+  </footer>
+
+    <style>
+      <!-- got bored of scrolling up all the way -->
+
+      #response-copied { display: none; }
+
+    </style>
+
+      <!-- Modal -->
+      <div class="modal fade" id="sendModal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel" aria-hidden="true">
+        <div class="modal-dialog">
+          <div class="modal-content">
+            <div class="modal-header">
+              <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+              <h4 class="modal-title"> प्रतिक्रिया भेजें।</h4>
```

Comment:
Like, https://github.com/netneutrality/savetheinternet.in/pull/146/files#diff-cfbc5c2e6c95e165f882116c822e785dR120
We should have `ट्राई को अपना जवाब भेजें`